index.html

```diff
@@ -16,30 +16,211 @@
+                    or website. Look through some of my work and experience! If you like what 
+                    you see and have a project you need coded, don’t hesitate to contact me.
+                </p>
+                <label class="connect">LET'S CONNECT</label>
+                <ul class="social-media-icons">
+                    <li>
+                        <a href="#" class="icons"><i class="fa-brands fa-twitter"></i></a>
+                    </li>
+                    <li>
+                        <a href="#" class="icons"><i class="fa-brands fa-linkedin"></i></a>
+                    </li>
+                    <li>
+                        <a href="#" class="icons"><i class="fa-sharp fa-solid fa-m"></i></a>
+                    </li>
+                    <li>
+                        <a href="#" class="icons"><i class="fa-brands fa-github"></i></a>
+                    </li>
+                    <li>
+                        <a href="#" class="icons"><i class="fa-solid fa-hand-peace"></i></a>
+                    </li>
+                </ul>
+                <button class="btn">Get my resume</button>
+
+                <div class="menu-list">
+                    <ul class="dropdown">
+                        <div class="open">
+                            <label class="dropdown-label">Languages</label>
+                            <i class="fa-solid fa-chevron-down arrow"></i>
+                        </div>
+                        <li>
+                            <div class="languages">
+                                <img src="./images/JavaScipt.png" alt="JavaScipt">
+                                <p>JavaScript</p>
+                            </div>
+                        </li>
+                        <li>
+                            <div class="languages">
+                                <img src="./images/HTML.png" alt="JavaScipt">
+                                <p>HTML</p>
+                            </div>
+                        </li>
+                        <li>
+                            <div class="languages">
+                                <img src="./images/CSS.png" alt="JavaScipt">
+                                <p>CSS</p>
+                            </div>
+                        </li>
+                    </ul>
+                    <ul class="dropdown">
+                        <div class="close">
+                            <label class="dropdown-label">Frameworks</label>
+                            <i class="fa-solid fa-chevron-right arrow"></i>
+                        </div>
+                        <hr>
+                    </ul>
+                    <ul class="dropdown">
+                        <div class="close">
+                            <label class="dropdown-label">Skills</label>
+                            <i class="fa-solid fa-chevron-right arrow"></i>
+                        </div>
+                        <hr>
+                    </ul>
```

Comment:
- [ ] Please inclose the technology categories ("Languages", "Frameworks" and "Skills") in a `ul`, and each of them in a` li` tags and then nested `list `tags inside these `li` tags as mentioned in the [project requirement ](https://github.com/microverseinc/curriculum-html-css/blob/main/portfolio/2_mobile.md#project--requirements)

> - Please use ul and li for categories in "Work" section. Html, Css, and Javascript should be list items.

> - Please use ul and li for categories in "About myself" section. Languages, Frameworks, and Skills should be list items.
 
e.g

```sh

<ul>
  <li>Languages
    <ul>
        <li>JavaScript</li>
        <li>HTML</li>
        <li>CSS</li>
    </ul>
  </li>
  <li>Frameworks</li>
  <li>Skills</li>
</ul>
```